playwright.config.ts

```diff
@@ -4,6 +4,8 @@ const { devices } = require('@playwright/test');
 
 const config = {
     forbidOnly: !!process.env.CI,
+    reporter: 'list',
+    workers: process.env.CI ? 6 : undefined,
```

Comment:
In the commit details, you said that using 6 workers causes 1-2 cases to time out. Do we want to set the number of workers to 6 then? Or should we set it to 4 workers until we find out why the overall time increases with more workers?

Reply:
Very good point. I do think it's better to use 4 workers as (1) it completes the smoke test run faster (2) showing ingestion lag at this point doesn't provide any value for web client's E2E behavior.

I will be updating the PR with 4 workers instead.

Reply:
Changed the number of workers to 4